src/ol/renderer/webgl/VectorTileLayer.js

```diff
@@ -98,7 +98,7 @@ class WebGLVectorTileLayerRenderer extends WebGLBaseTileLayerRenderer {
    */
   createRenderers_() {
     this.styleRenderers_ = this.styles_.map(
-      (style) => new VectorStyleRenderer(style, this.helper)
+      (style) => new VectorStyleRenderer(style, this.helper, false) // FIXME: implement
```

Comment:
I think these `FIXME` are a bit mysterious (I can't tell if they were meant to be addressed during development of this feature or if they are instructions to others).  If this is not going to be implemented in this branch, could you please remove the comment (or a more detailed comment might be easier for future readers to interpret - maybe "when we get around to implementing hit detection for the vector tile renderer, we will pass along an option from the constructor..." or something).